packages/ogmios/src/CardanoNode/queries.ts

```diff
@@ -94,8 +94,6 @@ export const queryEraSummaries = (client: LedgerStateQuery.LedgerStateQueryClien
 export const queryGenesisParameters = (client: LedgerStateQuery.LedgerStateQueryClient, logger: Logger) =>
   withCoreCardanoNodeError(async () => {
     logger.info('Querying genesis parameters');
-    // REVIEW: The queryNetwork/genesis local-state-query now expects one era as argument (either 'byron', 'shelley' or 'alonzo')
-    // to retrieve the corresponding genesis configuration.
-    // 'shelley' genesis maps best to the compact genesis we're using
+    // Update this to query multiple eras if the CompactGenesis type will have to include params from other eras
```

Comment:
Isn't this a `todo:`?

Reply:
No, but it does look like it. It's a hint regarding what to do in case we need genesis params from other eras.
But if you think it brings more confusion than clarification, I can remove it altogether.

Reply:
Ah I see, no it's fine.